spec/factories/task_files.rb

```diff
@@ -12,6 +12,13 @@
       fileable { build(:test) }
     end
 
+    trait :exportable do
+      internal_description { 'internal_description' }
+      usage_by_lms { 'display' }
+      used_by_grader { true }
+      visible { 'yes' }
+    end
+
     trait(:with_text_attachment) do
```

Comment:
In favor of consistency with the trait above. This also applies with the trait below.
```suggestion
    trait :with_text_attachment do
```